KakapoExample/KakapoTests/KakapoDBTests.swift

```diff
@@ -55,6 +55,64 @@ class KakapoDBTests: QuickSpec {
             beforeEach({
                 sut = KakapoDB()
             })
+            
+            it("should create a large number of elements") {
+                let queue = dispatch_queue_create("queue", DISPATCH_QUEUE_CONCURRENT)
+                dispatch_apply(1000, queue, { i in
+                    sut.create(UserFactory.self, number: 1)
+                })
+                
+                dispatch_apply(5000, queue, { i in
+                    sut.create(CommentFactory.self, number: 1)
+                })
+                
+                let userObjects = sut.findAll(UserFactory.self)
+                let user = sut.find(UserFactory.self, id: 1)
+                
+                let commentObjects = sut.findAll(CommentFactory.self)
+                let comment = sut.find(CommentFactory.self, id: 1002)
```

Comment:
how can you be sure there will be this comment? I don't think is a good idea to mix this 2 things. If comment is fast then a User could take the 1002 id